rust/common/error/mod.rs

```diff
@@ -101,7 +101,7 @@ error_messages! { TypeQLError
     InvalidAttributeTypeRegex { regex: String } =
         12: "Invalid regular expression '{regex}'.",
     GetVarRepeating { variable: Variable } =
-        13: "The variable '{variable}' occurred more than once in get query filter.",
+        13: "The variable '{variable}' occurred more than once in get query select.",
```

Comment:
`get` is obsolete - can you do a quick search for occurences please?

Reply:
Fixed. No other matches found.